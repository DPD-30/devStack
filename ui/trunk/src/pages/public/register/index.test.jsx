import React from 'react'
import { renderToString } from 'react-dom/server'

import PagesRegister from 'pages/public/register/index.jsx'

jest.unmock('pages/public/register/index.jsx')


describe('', () => {
    it('PagesRegister should match snapshot', () => {
        const comp = renderToString(<PagesRegister />)

        expect(comp).toMatchSnapshot()
    })
})
