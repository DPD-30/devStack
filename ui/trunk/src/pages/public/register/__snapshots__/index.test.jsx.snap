// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagesRegister should match snapshot 1`] = `
<div data-reactroot>
</div>
`;
